scripts/post_leap.sh

```diff
@@ -35,4 +35,11 @@ if [[ ! -f "${UPGRADE_LEAP_MARKER_FOLDER}/deploy-rpc.complete" ]]; then
 else
   log "deploy-rpc" "skipped"
 fi
+
+if [ "$UPGRADE_ELASTICSEARCH"="yes" ]; then
+    pushd /opt/rpc-upgrades/scripts
+        source ./elasticsearch_upgrade.sh
```

Comment:
This operation needs to be backgrounded as the ES index updates take upwards of 24h on normal deployments and we need to "finish" the main upgrade earlier

Reply:
Yeah, you're right about time consuming of ES reindex. And our original object is reduce interaction from user as less as possible, so automating this part as default. User can override es ugprade/redinex process at the beginning of leap now. Automated part is not mandatory.